### ISSUE TITLE
Add ability to tweak hyperparameters in learning rates

### ISSUE THREAD

**Issue by dustinvtran** [airoldilab/sgd#70]
It should be a list of arguments, since each learning rate can have a different number of hyperparameters. fb5f2cd7d9f43c30c15290fad4631888641f16e2 makes progress on this but only works for a single hyperparameter for `one-dim`
